recipe detail  customization (#31)
miamtech/mealz-documentation: docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailCustomization.mdx

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import Header from './RecipeDetailHeader.mdx';
import Info from './RecipeDetailInfo.mdx';
import SponsorBanner from "./RecipeDetailSponsorBanner.mdx";
import Tags from "./RecipeDetailTags.mdx"
import Swapper from "./RecipeDetailSwapper.mdx"
import Footer from "./RecipeDetailFooter.mdx"
import ProductListHeader from "./RecipeDetailProductListHeader.mdx"
import Product from "./RecipeDetailProduct.mdx"
import OftenDeleted from "./RecipeDetailOftenDeleted.mdx"
import Unavailable from "./RecipeDetailUnavailable.mdx"
import Ingredients from "./RecipeDetailIngredients.mdx"
import Steps from "./RecipeDetailSteps.mdx"

## RecipeDetail

> RecipeDetails is one of the most customizable component, It has two customizations state **success** & **loading:**

Reviewer: ```suggestion
RecipeDetails is one of the most customizable components, it has two customization states: **success** & **loading:**
```

### Loading 

> Loading is of type [**LoaderConfig**](../../base-component/customize-views#loading) you can eather create a specific 
> template for RecipeDetails otherwise the default template define in **defaultViews** would be use.

Reviewer: ```suggestion
Loading is of type [**LoaderConfig**](../../base-component/customize-views#loading). You can create a custom 
template for RecipeDetails or otherwise the default template defined in **defaultViews** will be used.
```

> `LoaderParameters` is an empty object for now, but tomorrow we'll be able to add properties to it, in order to avoid breaking changes in future versions. 

Reviewer: ```suggestion
`LoaderParameters` is an empty object for now, but we have the option to add properties to it in the future in order to avoid breaking changes. 
```

```kotlin
object LoaderParameters
```

### Success

> Success state containe 12 customizable slots and one configuration field :

Reviewer: ```suggestion
Success state contains 12 customizable slots and one configuration field :
```

<img src="https://storage.googleapis.com/assets.miam.tech/kmm_documentation/android/customization/recipe-detail-1.png" alt="templating 1" />
<img src="https://storage.googleapis.com/assets.miam.tech/kmm_documentation/android/customization/recipe-detail-2.png" alt="templating 2" />
<img src="https://storage.googleapis.com/assets.miam.tech/kmm_documentation/android/customization/recipe-detail-3.png" alt="templating 3" />

### Full customisation setting

> if you want to fully customize this component we remande to copy that code into your mealz template configuration.
> You can remove useless part

Reviewer: ```suggestion
If you want to fully customize this component we recommend copying the code into your Mealz template configuration.
You can remove the code you do not overwrite.
```
